src/ui/server/controllers/insurance/policy/name-on-policy/validation/rules/position.ts

```diff
@@ -1,7 +1,7 @@
 import INSURANCE_FIELD_IDS from '../../../../../../constants/field-ids/insurance';
 import { ERROR_MESSAGES } from '../../../../../../content-strings';
 import { POLICY_FIELDS } from '../../../../../../content-strings/fields/insurance/policy';
-import alphaCharactersAndMaxLengthValidation from '../../../../../../shared-validation/alpha-characters-and-max-length';
+import nameValidation from '../../../../../../shared-validation/name';
```

Comment:
Is this expected, as IMO i'd say "position" is not a name field?

Reply:
it uses the same validation.  Would be duplication if I made one for position